lib/singleeyefitter/projection.h

```diff
@@ -259,7 +261,7 @@ namespace singleeyefitter {
                 auto sgnl = sign(l);
                 T3 << 0, -n*sign(l), l,
                     sign(l), 0, 0,
-                    0, abs(l), n;
+                    0, std::abs(l), n;
```

Comment:
Nit: can you change this to a "using std::abs" to allow ADL? I know it's overkill, but it's for consistency.

Reply:
Sure. Just to make sure I got the ADL part since I wasn't aware of the term before: The rationale is that we allow the compiler to search for an abs function in the namespace where the Scalar is to be used?

Reply:
Correct, for example to find ceres::abs if performing forward differentiation (cf. https://github.com/ceres-solver/ceres-solver/issues/283)